client/components/ModulesTreemap.jsx

```diff
@@ -18,11 +18,17 @@ import Search from './Search';
 import {store} from '../store';
 import ModulesList from './ModulesList';
 
-const SIZE_SWITCH_ITEMS = [
-  {label: 'Stat', prop: 'statSize'},
-  {label: 'Parsed', prop: 'parsedSize'},
-  {label: 'Gzipped', prop: 'gzipSize'}
-];
+function allSizeSwitchItems() {
+  const items = [
+    {label: 'Stat', prop: 'statSize'},
+    {label: 'Parsed', prop: 'parsedSize'}
+  ];
+
+  if (window.compressionAlgorithm === 'gzip') items.push({label: 'Gzipped', prop: 'gzipSize'});
```

Comment:
I would change `gzipSize/brotliSize ` to `compressedSize` but looks like it will be a breaking change.

Reply:
Would be nice if we could first do this as a new feature, release in a minor version and then do a major release later to consolidate the API on on plain `compressedSize`. That has been my opinion during this PR review at least ☺️.

What do you think of this plan?